CS/DataContext_Ex/MainWindow.xaml

```diff
@@ -1,96 +1,117 @@
         <dxd:DockLayoutManager.DataContext>
             <local:MyDescriptionClass Description="This example demonstrates how to bind layout elements to binding sources" />
         </dxd:DockLayoutManager.DataContext>
-
         <dxd:LayoutGroup Orientation="Vertical">
-            <dxd:LayoutPanel x:Name="panelDescription" Caption="Description" ItemHeight="70">
+            <dxd:LayoutPanel
+                x:Name="panelDescription"
+                Caption="Description"
+                ItemHeight="70">
                 <dxd:LayoutGroup Orientation="Vertical">
-                    <!--Define a label and bind its caption to MyDescriptionClass.Description.
-                    A MyDescriptionClass object is assigned to the DockLayoutManager.DataContext, and this DataContext is inherited by all layout items.
-                    Since the label is not a FrameworkElement, the binding statement must be created using the explicit syntax.
+                    <!--
+                        Define a label and bind its caption to MyDescriptionClass.Description.
+                        A MyDescriptionClass object is assigned to the DockLayoutManager.DataContext, and this DataContext is inherited by all layout items.
+                        Since the label is not a FrameworkElement, the binding statement must be created using the explicit syntax.
                     -->
                     <dxd:LabelItem Caption="{Binding Path=DataContext.Description, RelativeSource={RelativeSource Mode=Self}}" />
                 </dxd:LayoutGroup>
             </dxd:LayoutPanel>
-
             <dxd:LayoutGroup Orientation="Horizontal">
-
-                <!--The panel that has its own DataContext-->
+                <!--  The panel that has its own DataContext  -->
                 <dxd:LayoutPanel x:Name="panelPerson1" Caption="Person 1">
-                    <!--Provide DataContext for this panel and its child controls-->
+                    <!--  Provide DataContext for this panel and its child controls  -->
                     <dxd:LayoutPanel.DataContext>
-                        <local:Person Title="Mr" FirstName="David" LastName="Lawrence" Country="USA" City="Denver" HomePhone="303-555-5555" />
+                        <local:Person
+                            Title="Mr"
+                            City="Denver"
+                            Country="USA"
+                            FirstName="David"
+                            HomePhone="303-555-5555"
+                            LastName="Lawrence" />
                     </dxd:LayoutPanel.DataContext>
-                    <!--Bind to properties on the DataContext. The DataContext is inherited from panelPerson1.
-                    An implicit syntax for creating binding statements can be used, as TextBox controls are FrameworkElements.
+                    <!--
+                        Bind to properties on the DataContext. The DataContext is inherited from panelPerson1.

```

Comment:
`Bind to properties on the DataContext.`

Is "on" the correct preposition in this technical context?

Reply:
```suggestion
                        Binds to properties from the DataContext that is inherited from panelPerson1.

```